app/server/routes/community.js

```diff
@@ -44,10 +56,14 @@ router.post("/", async (req, res) => {
   }
 });
 
-// Get all communities
+// Get all communities you are a part of
 router.get("/", async (req, res) => {
   try {
-    const communities = await Community.find({}, "", {
+    const user = await User.findOne({ name: req.user.name });
+
+    if (!user) return res.status(404).send("User not found.");
```

Comment:
Just thinking out loud. I can see why the 404 is used here, but would a 403 also work here?

Reply:
sorry not 403. 401 unauthorized.

Reply:
So 404 = Not found, 401 = Not authorized, and 403 is Forbidden. In this case the user isn't found in the database with the search condition of req.user.name which comes from the access token. So yes technically it could be any one of these, the question is just what best describes the error. I think I was thinking more of user not being found in the database as in 404, but it could be argued that its not found because the access token doesn't exist anymore like after it expires, meaning they would be unauthorized.